en/firefox/products.ftl

```diff
@@ -4,63 +4,70 @@
-firefox-products-ios = { -brand-name-ios }
-firefox-products-see-all-browsers = See all browsers
 firefox-products-mozilla-vpn = { -brand-name-mozilla-vpn }
 firefox-products-surf-stream-and-get-work-done = Surf, stream and get work done on servers in over 30 countries for a secure internet connection with a new perspective.
 firefox-products-get-mozilla-vpn = Get { -brand-name-mozilla-vpn }
-firefox-products-relay = { -brand-name-firefox-relay}
-firefox-products-protect-your-real = Protect your real email address to help control your inbox.
-firefox-products-learn-more-about-relay = Learn more about { -brand-name-firefox-relay }
 
-firefox-products-send-your-large-files-and = Send your large files and sensitive documents safely, up to 2.5G.
-firefox-products-send-a-file = Send a file
-firefox-products-pocket = { -brand-name-pocket }
-firefox-products-discover-the-best-content = Discover the best content on the web — and consume it wherever and whenever you want.
-firefox-products-get-pocket = Get { -brand-name-pocket }
-firefox-products-learn-more-about-pocket = Learn more about { -brand-name-pocket }
+## MDN Plus
 
-# Variables:
-#   $fxa_attr (string) - anchor link url and attributes
-#   $accounts_attr (string) - anchor link url and attributes
-firefox-products-already-have-an-account-sign = Already have an account? <a { $fxa_attr }>Sign In</a> or <a { $accounts_attr }>learn more</a> about joining { -brand-name-firefox }.
+firefox-products-mdn-plus = { -brand-name-mdn-plus }
+firefox-products-resources-for-developers = Resources for developers, by developers. Support { -brand-name-mdn } and make it your own with collections, notifications, and playgrounds.
+firefox-products-support-mdn = Support { -brand-name-mdn }
 
-firefox-products-firefox-focus = { -brand-name-firefox-focus }
-firefox-products-your-dedicated-privacy = Your dedicated privacy browser with automatic tracking protection and ad blocking.
+## Thunderbird
+
+firefox-products-thunderbird = { -brand-name-thunderbird }
+firefox-products-access-all = Access all your email, calendars, and contacts in one fast app. Filter and organize them the way you like.
+firefox-products-download-thunderbird = Download { -brand-name-thunderbird }
+
+## Fakespot
+
+firefox-products-fakespot = { -brand-name-fakespot }
+# scam artists are people who are trying to sell fakes or steal money
+# 'weeding' is a gardening term, here it is used to mean "removing unwanted things without touching the good ones"
+firefox-products-fakespot-has-your = { -brand-name-fakespot } has your back when you’re shopping online — weeding out scam artists and finding you the best deals. Made by { -brand-name-mozilla }.
+firefox-products-analyze = Analyze a URL
+
+## Pocket
+
+firefox-products-pocket = { -brand-name-pocket }
+firefox-products-discover-the-best-content = Discover the best content on the web — and consume it wherever and whenever you want. Made by { -brand-name-mozilla }.
```

Comment:
GitHub's diff structure made me miss that this is modifying an existing string :-( 

Which means we now have a ton of errors across locales.

Opened https://github.com/mozilla/bedrock/pull/14555